src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -53,6 +53,13 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
   );
   const purchaseHistory = useSelector(userPurchaseHistorySelector);
 
+  const redirectUrl = venue?.config?.redirectUrl ?? "";
+  const { hostname } = window.location;
+
+  if (redirectUrl && redirectUrl !== hostname) {
```

Comment:
What value gets set in the DB for this? I would have expected to see code here that checks new URL(valueFromDB).hostname or similar. Have we tested that this works as expected?

Reply:
And I guess if the value in the DB is just a hostname, then calling it ‘url’ seems misleading/unintuitive.